### PR TITLE
feat(config): add namron 4 channel and fix 1+2 channel switch

### PR DESCRIPTION
**Pull request by Chr1stian**
Just let me know if you want anything more added to these switches. They are all part of OZW, but the import command did not work for me.

### CODE REVIEW

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Please double-check the style guide. `supportsZWavePlus` is no longer used and should be removed. And the device label/description seems off:

grafik

**Comment by Chr1stian** [comment]
> Please double-check the style guide. `supportsZWavePlus` is no longer used and should be removed. And the device label/description seems off:
> 
> grafik

See new commit. Is this good?
The product(s) can be seen here https://products.z-wavealliance.org/products/3639 and here https://www.elektroimportoren.no/namron-z-wave-4-kanaler-bryter-k8/4512714/Product.html

File: packages/config/config/devices/0x0438/1_kanal_bryter.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
 {
 	"manufacturer": "Namron AS",
 	"manufacturerId": "0x0438",
-	"label": "1 KANAL BRYTER",
-	"description": "",
+	"label": "7070990327756",
+	"description": "Namron 1 kanal bryter K2",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0438/1_kanal_bryter.json]: The label should be friendlier if possible as it appears in the UIs. I'd make the Namron 1 the label and the rest the description. Please carry this through to the other files.

Also, per the style guide the description should be Title Case. Capitilize each word.

File: packages/config/config/devices/0x0438/1_kanal_bryter.json [hunk 2]
```diff
@@ -15,6 +15,17 @@
...
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Button 1",
+			"maxNodes": 5
+		}
+	},
 	"paramInformation": {
 		"1": {
 			"label": "Factory Reset",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0438/1_kanal_bryter.json:32]: Remove the unnecessary description per the style guide. Please take a look at the other files and carry through.

File: packages/config/config/devices/0x0438/1_kanal_bryter.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
 {
```
Comment (blhoward2) [inline packages/config/config/devices/0x0438/1_kanal_bryter.json:3]: The comment at the top does not fit the style guide. Carry to other files. See the style guide at Comments. https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide

**Review by blhoward2 — CHANGES_REQUESTED** [review]
Please take a look at my comments and the style guide and correct.

**Comment by Chr1stian** [comment]
> Please take a look at my comments and the style guide and correct.

Looked more closely at the styleguide and updated. It says to remove to brand name from both label and description, so I did that, but otherwise followed your suggestions. Please review :)

File: packages/config/config/devices/0x0438/1_kanal_bryter.json (hunk 2 above)
Comment (Chr1stian) [reply packages/config/config/devices/0x0438/1_kanal_bryter.json:32]: What is the unnecessary description you are talking about here @blhoward2 ?

**Review by blhoward2 — DISMISSED** [review]
I just went ahead and made the changes. The description I meant is "Factory Reset" under a parameter labeled "Factory Reset." It adds nothing.

**Comment by Chr1stian** [comment]
> I just went ahead and made the changes. The description I meant is "Factory Reset" under a parameter labeled "Factory Reset." It adds nothing.

Thank you!

File: packages/config/config/devices/0x0438/k8.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Factory Reset",
+			"valueSize": 2,
+			"minValue": 21930,
+			"maxValue": 21930,
+			"defaultValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0438/k8.json]: That doesn't make sense. And we usually use options for a factory reset (also applies to the other files).

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0438/k8.json (hunk 4 above)
Comment (Chr1stian) [reply packages/config/config/devices/0x0438/k8.json]: The manual says "set to 21930 to factory reset". What options do you usually use for factory reset? If you could link an example I will fix it for these files 🙂

Comment (AlCalzone) [reply packages/config/config/devices/0x0438/k8.json]: I meant the default value needs to be in the range defined by min/max - we should probably add an automated check for that. Let me see if I find one...

Comment (AlCalzone) [reply packages/config/config/devices/0x0438/k8.json]: Here's one:
https://github.com/zwave-js/node-zwave-js/blob/32b5f9fc76afc21439d5f88b534cb3654156612b/packages/config/config/devices/0x0090/templates/kwikset_template.json#L258-L279

**Comment by Chr1stian** [comment]
@zwave-js-bot fix lint

**Review by blhoward2 — APPROVED** [review]

**Comment by Chr1stian** [comment]
Will you check the changes requested @AlCalzone ?

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**